README.md

```diff
@@ -1 +1,38 @@
 # hw01
+## 10/04/2020
+***
+
+Hi everyone, my name is Emily Robb and I'm a fourth year in the College. I'm double-majoring in Philosophy and Political Science and am incredibly excited to take this course and learn practical skills that I haven't really had previous exposure to. Looking forward to learning with all of you!
```

Comment:
Your assignment is great - the only thing I'd add is maybe more information about _you_ (Adding your interests, where you're from etc.)!